.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description
+[Please provide general introduction to the issue/proposal.]
+
+
+## Reproduction
+[If possible, please include example code and image(s) to reproduce this Issue.]
+
+
+## Environment
+[If reporting a bug, please include the following important infomation about your system.]
+- [ ] Operating system and its version
+- [ ] Python version (run `sys.version`)
+- [ ] scikit-image version (run `skimage.__version__`)
+
+
+## Traceback
```

Comment:
Can we instruct people to include this in the description instead?  I'm trying to shorten this template.